demos/python_demos/whiteboard_inpainting_demo/utils/network_wrappers.py

```diff
@@ -0,0 +1,123 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import json
+import logging as log
+
+import cv2
+import numpy as np
+
+from utils.ie_tools import IEModel
+from .segm_postprocess import postprocess
+
+
+class MaskRCNN(IEModel):
+    def __init__(self, ie, model_path, conf=.6, device='CPU', ext_path='', max_reqs=1):
+        super().__init__(ie, model_path, conf, device, ext_path, max_reqs)
+
+        required_input_keys = [{'im_info', 'im_data'}, {'im_data', 'im_info'}]
+        current_input_keys = set(self.inputs_info.keys())
+        assert current_input_keys in required_input_keys
+        required_output_keys = {'boxes', 'scores', 'classes', 'raw_masks'}
+        assert required_output_keys.issubset(self.net.outputs)
+
+        self.n, self.c, self.h, self.w = self.inputs_info['im_data'].shape
+        assert self.n == 1, 'Only batch 1 is supported.'
```

Comment:
```suggestion
```
You've set batch to one in `__init__` of super.

Reply:
Fixed.